test/unit/sdam/server_selection_spec_helper.js

```diff
@@ -1,147 +1,17 @@
 'use strict';
-const path = require('path');
-const fs = require('fs');
-const { Topology } = require('../../../../src/sdam/topology');
-const { Server } = require('../../../../src/sdam/server');
-const { ServerType, TopologyType } = require('../../../../src/sdam/common');
-const { ServerDescription } = require('../../../../src/sdam/server_description');
-const { ReadPreference } = require('../../../../src/read_preference');
-const { MongoServerSelectionError } = require('../../../../src/error');
-const ServerSelectors = require('../../../../src/sdam/server_selection');
-
-const { EJSON } = require('bson');
+const { Topology } = require('../../../src/sdam/topology');
+const { ServerType, TopologyType } = require('../../../src/sdam/common');
+const { ServerDescription } = require('../../../src/sdam/server_description');
+const { ReadPreference } = require('../../../src/read_preference');
+const { MongoServerSelectionError } = require('../../../src/error');
```

Comment:
Does it make sense to move this to `tools/spec-runner`? And/or rename it from `helper` -> `runner`?

Reply:
possibly, but there's some standardization overall that we can make for things like this which will be more evident when we do the full spec audit (up next after finishing the general sorting phase)

Reply:
Yea makes sense, kinda goes along with my other comment about the custom loader functions, we'll defer to later